src/components/screens/NewsItem.js

```diff
@@ -63,7 +63,7 @@ export const NewsItem = ({ data, route }) => {
       <WrapperWithOrientation>
         {!!title && !!link ? (
           <TitleContainer>
-            <Touchable onPress={openWebScreen}>
+            <Touchable navigation={navigation} onPress={openWebScreen}>
               <Title accessibilityLabel={`${trimNewLines(title)} ${a11yText.heading}`}>
```

Comment:
play around with navigation , thinking that it carries a share prop. Probably I just misunderstood a conversation with you @digorath ...